deploy-teuthology

```diff
@@ -173,19 +173,17 @@ fi
 
 if $SETUPNS ; then
 ZONE_DOMAIN=${OS_CLOUD}.suse.de
-DNSKEY_NAME=$(make_dns_key $ZONE_DOMAIN $PWD/secrets)
-PUBKEY_NAME=${DNSKEY_NAME}.key
-PRIKEY_NAME=${DNSKEY_NAME}.private
-ZONE_SECRET=$(cat $PWD/secrets/$PRIKEY_NAME | yj | jq -r .Key)
+# dnssec key generation was temporarily moved to 'teuthology_nameserver' ansible role due to encryption algorithm deprication on Tumbleweed
+#DNSKEY_NAME=$(make_dns_key $ZONE_DOMAIN $PWD/secrets)
+#PUBKEY_NAME=${DNSKEY_NAME}.key
+#PRIKEY_NAME=${DNSKEY_NAME}.private
+#ZONE_SECRET=$(cat $PWD/secrets/$PRIKEY_NAME | yj | jq -r .Key)
 cat >> $STATE_PATH/teuthology_server.yml << END
 #keys_dir: $STATE_PATH
 nsupdate_web_server: "$private_addr"
 lab_domain:  $ZONE_DOMAIN
 zone_name:   $ZONE_DOMAIN
 zone_domain: $ZONE_DOMAIN
-zone_secret: $ZONE_SECRET
-pubkey_name: $PUBKEY_NAME
-prikey_name: $PRIKEY_NAME
 END
 scp $sshopts $PWD/secrets/${DNSKEY_NAME}* $LOGIN_USER@$target:
```

Comment:
This code is senseless now, since ansible is generating the keys.

Reply:
yep I missed to remove this line